test/test.c

```diff
@@ -462,6 +462,41 @@ static void test_unix_permissions(void) {
   remove(RFILE);
   remove(ZIPNAME);
 #endif
+ }
+
+static void test_extract_stream(void) {
+
+  remove(ZIPNAME);
+
+  struct zip_t *zip = zip_open(ZIPNAME, ZIP_DEFAULT_COMPRESSION_LEVEL, 'w');
+  assert(zip != NULL);
+
+  assert(0 == zip_entry_open(zip, RFILE));
+  assert(0 == zip_entry_write(zip, TESTDATA1, strlen(TESTDATA1)));
+  assert(0 == zip_entry_close(zip));
+
+  zip_close(zip);
+
+  remove(RFILE);
+
+  FILE *fp = NULL;
+  fp = fopen(ZIPNAME, "r+");
+  assert(fp != NULL);
+ 
+  fseek(fp, 0L, SEEK_END);
+  size_t filesize = ftell(fp);
+  fseek(fp, 0L, SEEK_SET);
+
+  char stream[filesize];
+  memset(stream, 0, filesize);
+  size_t size = fread(stream, 1, filesize, fp);
```

Comment:
just in case replace `1` by `sizeof(char)`

Reply:
ok.